test-requirements.txt

```
@@ -1 +1,2 @@
 odoo_test_helper
+odoo-addon-sale_fixed_discount @ git+https://github.com/OCA/account-invoicing.git@refs/pull/1488/head#subdirectory=setup/sale_fixed_discount
```

```suggestion
odoo-addon-account-invoice-fixed-discount @ git+https://github.com/OCA/account-invoicing.git@refs/pull/1488/head#subdirectory=setup/account_invoice_fixed_discount
```

@PieterPaulussen This line is intended to install the module that is not yet merged. So, account_invoice_fixed_discount.

---

@PieterPaulussen you still need to adapt the module name at the end of the string. I think you only changed the beginning. 

Good work so far.. looking forward to do the testing/review.

---

~~I've refactored the flow of this module to align it with the new implementation of `account_invoice_fixed_discount` and the option arises to remove the requirement of the module entirely as both features can be used as stand-alone. The only problem is that the `discount_fixed` value does not get updated when invoicing the order.
Perhaps you know of a better way like an optional dependency or a new module entirely?~~